packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -38,6 +39,9 @@ export default util.createRule({
             type: 'method definition',
             name: util.getNameFromPropertyName(methodDefinition.key),
           },
+          fix(fixer) {
+            return [fixer.insertTextBefore(methodDefinition, 'public ')];
```

Comment:
I believe you can just return the fixer action here.
Also, if there are going to be multiple fixes created, IMO we should use a generator function instead.

Reply:
@j-f1 Thank you for the review.

> I believe you can just return the fixer action here.

I agree. I'll update this.

> Also, if there are going to be multiple fixes created, IMO we should use a generator function instead.

Could you tell me a URL to the rule implementation which utilizes a generator function for fixer? I have not seen a generator function is used for fixer. And also could not find doc for it in [API doc](https://eslint.org/docs/developer-guide/working-with-rules-deprecated#applying-fixes).

Reply:
That’s the deprecated API doc, although it’s helpfully not labeled as such. Check out [the current one](https://eslint.org/docs/developer-guide/working-with-rules#applying-fixes) which says you can return a generator.

Reply:
For example, #289.

Reply:
@j-f1 @mysticatea Thank you for the pointers. I read a wrong documentation.

I agree that we should use `*fix(fixer)` instead. But it requires update for type definition and it will conflict with #289 changes and I want to avoid that.

1. Wait #289 is merged, rebase this branch upon master and fix the point
2. Merge this branch as-is and when #289 is merged, I'll create another PR to fix the point

I feel either would be fine. @mysticatea, do you predict #289 will be merged soon?

Reply:
Since this is just one fix, you should be fine without the generator.

Reply:
@j-f1 

Sure. I'll only work on following comment tonight.

> I believe you can just return the fixer action here.

Reply:
Great 👍